### PR TITLE
Adds page title of form subtitle - title - WMT to all pages

### PR DESCRIPTION
**Pull request by tsheppard01**

### CODE REVIEW

File: app/views/includes/page-title.html [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{% if subTitle %}{{ subTitle }} - {% endif %}
+{% if title %}{{ title }}{% endif %} 
+- WMT
```
Comment (caoimheb) [inline app/views/includes/page-title.html]: Maybe include the `-` in the second `if`. Then if there is no subtitle or title the heading won't be `- WMT`, but `WMT`?

Comment (caoimheb) [inline app/views/includes/page-title.html]: We not including any detail about the tab they are on, e.g. `National - HMPPS - Capacity`?

Comment (tsheppard01) [reply app/views/includes/page-title.html]: Andy answered this F2F

Comment (tsheppard01) [reply app/views/includes/page-title.html]: Good spot, fixed

**Pull request merged**